packages/SwingSet/src/kernel/initializeKernel.js

```diff
@@ -44,12 +44,12 @@ export function initializeKernel(config, hostStorage, verbose = false) {
       logStartup(`adding vat '${name}' from bundle ${bundleName}`);
 
       if (bundle) {
-        if (typeof bundle !== 'object') {
-          throw Error(`bundle is not an object, rather ${bundle}`);
-        }
-      } else if (!bundleName) {
-        throw Error(`no bundle specified for vat ${name}`);
-      }
+        assert.typeof(
+          bundle,
+          'object',
+          X`bundle is not an object, rather ${bundle}`,
+        );
+      } else assert(bundleName, X`no bundle specified for vat ${name}`);
```

Comment:
Ooh, a bare `else`! No curlies to hide it! Correct but risque. I secretly like it :)

Reply:
Oops. I fixed it. Your secret is safe with me.